src/Fitter.cc

```diff
@@ -169,20 +172,28 @@ Int_t Fitter::fit()
     m.setStrategy(2);
     m.migrad() ;
     m.hesse() ;
-    m.minos() ;
+    // m.minos() ;
```

Comment:
is this intended?

Reply:
Yes, the error is now computed using the dedicated method, [MinosAng](https://github.com/CMSKStarMuMu/UML-fit/blob/fit-data-controlRegion-merged/src/Fitter.cc#L331). In case of fit with large statistics, like full MC sample or control region, we don't want to run MINOS by default, since it could increase a lot the running time